.github/workflows/publish.yaml

```diff
@@ -29,6 +29,9 @@ jobs:
     permissions:
       id-token: write
       contents: read
+    strategy:
+      matrix:
```

Comment:
Since we are using a matrix setup for testing multiple versions, do you think we should add the `fail-fast` option? 

```yaml
strategy:
  fail-fast: false
```